app/Models/Signup.php

```diff
@@ -137,6 +137,11 @@ public function toBlinkPayload()
      */
     public function getQuantity()
     {
+        if (! config('features.v3QuantitySupport')) {
+            return $this->quantity;
```

Comment:
I _think_ we still want to do the logic around checking if there is a `quantity_pending` first and return that if `quantity` is `null` (below in lines 155-161)?

Reply:
ah yeeepppp!!! i always forget about `quantity_pending`